pkg/controller/operands/kubevirt.go

```diff
@@ -300,6 +354,13 @@ func translateKubeVirtConds(orig []kubevirtv1.KubeVirtCondition) []conditionsv1.
 }
 
 func NewKubeVirtConfigForCR(cr *hcov1beta1.HyperConverged, namespace string) *corev1.ConfigMap {
+	const constFeatureGates = "DataVolumes,SRIOV,LiveMigration,CPUManager,CPUNodeDiscovery,Sidecar,Snapshot"
```

Comment:
You are right. However, since we didn't remove the CM yet, this logic keep the old FGs with the old behaviour (for backward compatiblity), and currenly only the `HotplugVolumes` is controlled by HCO.